src/mesh/ProtobufModule.h

```diff
@@ -61,6 +62,14 @@ template <class T> class ProtobufModule : protected SinglePortModule
         return sender;
     }
 
+    int handleStatusUpdate(const meshtastic::Status *arg)
+    {
+        if (arg->getStatusType() == STATUS_TYPE_NODE) {
+            numOnlineNodes = nodeStatus->getLastNumTotal();
```

Comment:
Shouldn't this be `getNumOnline()`?

Reply:
Yuuuup. I just ran it locally and thought my numbers looked odd. 😅